### ISSUE TITLE
'C# Interactive' window is editable by dragging the selection

### ISSUE THREAD

**Issue by rverberne** [lukebuehler/CShell#58]
You can select text in the C# interactive window and drag it to another line, creating a mangled mess. Not HUGE deal but it should be readonly right?

**Comment by lukebuehler** [comment]
yes, you are right, just tested it, ugly! But should be easy to fix it.
